ZadanieZespolowe1.Rmd

```diff
@@ -22,10 +22,43 @@ bezrobocie_typ <- "b"
 Celem poniższego dokumentu jest analiza danych dotyczących stopy bezrobocia w Polsce na podstawie danych GUS i prognoza wysokości stopy na rok `r rok`.
 
 ## Wczytanie danych
+Dane zostały wczytane i odpowiednio zmodyfikowane w oddzielnym R skrypcie. Tabela została wyodrębniona ze strony z usunięciem niepotrzebnych znaków, ujednoliceniem separatora dziesiętnego i dodaniem możliwości wyboru wariantu w roku 2002:
+
+w wariancie "a"- dane według wyników z Powszechnego Spisu Rolnego 1996,
+
+w wariancie "b"- dane według wyników z Narodowego Spisu Powszechnego Ludności i Mieszkań oraz Powszechnego Spisu Rolnego 2002
+
+W naszych rozważaniach domyślnie weźmiemy pod uwagę wariant "b".
+
+Tabela została również przekształcona do postaci długiej. W ten sposób otrzymaliśmy ramkę danych złożoną z czterech kolumn- Rok, Miesiąc, Wartość i MiesiącInt (miesiąc w postaci liczby całkowitej)
+
 ```{r}
 dane <- readRDS(paste0("dane/bezrobocie_",
                        bezrobocie_typ,
                        ".rds"))
+tail(dane)
+```
+## Podstawowe wykresy
+Zdefiniujmy szereg czasowy opisujący nasze zjawisko i narysujmy dla niego podstawowe wykresy
+```{r}
+bezrobocie <- ts(dane$Wartość[length(dane$Wartość):1])
```

Comment:
Tu ja dałem dupy, niepotrzebnie tabelka z danymi przy pobieraniu jest odwrócona kolejnością, przez to tutaj znowu trzeba odwracać. Zrobię commita do tamtego kodu. A na przyszłość łatwiej odwracać przez funkcję `rev()` 😄

Problem z tym szeregiem czasowym jest chwilowo taki, że nic on za bardzo nam nie wnosi. Dobrze by było, żebyśmy mieli tu też ustawione parametry `frequency`, `start` i `end`. Od razu doszedłby jeden parametr globalny dla kodu na tą częstotliwość (opcja na sezonowość miesięczną, kwartalną itd.).